Attributes for all mentions of kubernetes, openshift, namespace, project. + Whilelabel playbook (#1602)
codenvy/che-docs: modules/installation-guide/partials/proc_using-minikube-to-set-up-kubernetes.adoc

// installing-{prod-id-short}-on-minikube

[id="using-minikube-to-set-up-kubernetes_{context}"]

> = Using Minikube to set up {prod-uptream-name}

Reviewer: ```suggestion
= Using Minikube to set up {kubernetes}
```

> This section describes how to use Minikube to set up a local single-node {prod-uptream-name} cluster.

Reviewer: ```suggestion
This section describes how to use Minikube to set up a local single-node {kubernetes} cluster.
```

.Prerequisites

* An installation of `kubectl`. See link:https://kubernetes.io/docs/tasks/tools/install-kubectl/[Installing and Setting Up kubectl].

> * An installation of Minikube with {prod-uptream-name} version `1.9` or higher. See link:https://kubernetes.io/docs/tasks/tools/install-minikube/[Installing Minikube].

Reviewer: ```suggestion
* An installation of Minikube with {kubernetes} version `1.9` or higher. See link:https://kubernetes.io/docs/tasks/tools/install-minikube/[Installing Minikube].
```
